sdk/authorization/microsoft-azure-authentication-msi-token-provider/README.md

```diff
@@ -1,70 +0,0 @@
-# What is this?
```

Comment:
We should keep the README to just add a deprecation disclaimer as we have in this [Event Grid](https://github.com/Azure-Samples/event-grid-java-publish-consume-events/blob/master/README.md) README and include a link to the migration guide. Another advantage of README is that it also shows the markdown on the landing page, so, it's immediately visible.